src/tcgwars/logic/impl/gen4/Platinum.groovy

```diff
@@ -846,9 +846,14 @@ public enum Platinum implements LogicCardInfo {
             energyCost C
             onAttack {
               damage 20
-              turn flag = bg.turnCount + 2
+              applyAfterDamage(ASLEEP)
+
+              turnFlag = bg.turnCount + 2
               pokeFlag = true
```

Comment:
You also have to make sure it doesn't allow Metronome: Midnight Eyes to set the effect apparently because you are _actually_ being hit by Metronome in that case.